airflow/models/dag.py

```diff
@@ -686,7 +689,11 @@ def handle_callback(self, dagrun, success=True, reason=None, session=None):
             ti.task = self.get_task(ti.task_id)
             context = ti.get_template_context(session=session)
             context.update({'reason': reason})
-            callback(context)
+            try:
+                callback(context)
+            except Exception:
+                self.log.exception("failed to invoke dag state update callback")
+                Stats.incr("dag.callback_exceptions")
```

Comment:
Can you add this key to docs? 
https://airflow.readthedocs.io/en/latest/metrics.html

Reply:
good call, i always forgot about that doc page :)